packages/grafana-runtime/src/services/contextSrv.ts

```diff
@@ -0,0 +1,51 @@
+export interface User {
+  id: number;
+  isGrafanaAdmin: any;
+  isSignedIn: any;
+  orgRole: any;
+  orgId: number;
+  orgName: string;
+  login: string;
+  orgCount: number;
+  timezone: string;
+  helpFlags1: number;
+  lightTheme: boolean;
+  hasEditPermissionInFolders: boolean;
+  email?: string;
+}
+
+export interface ContextSrv {
+  pinned: any;
+  version: any;
+  user: User;
+  isSignedIn: any;
+  isGrafanaAdmin: any;
+  isEditor: any;
+  hasEditPermissionInFolders: boolean;
+  minRefreshInterval: string;
+  setLoggedOut: () => void;
+  hasRole: (role: string) => boolean;
+  isGrafanaVisible: () => boolean;
+  isAllowedInterval: (interval: string) => boolean;
+  getValidInterval: (interval: string) => string;
+  hasAccessToExplore: () => boolean;
+}
```

Comment:
Please add [tsdoc](https://tsdoc.org/) annotations. See BackendSrv for a reference: https://github.com/grafana/grafana/blob/a51cc050e2df5f7a58c18b0fda9ae6977bf7558f/packages/grafana-runtime/src/services/backendSrv.ts#L3